components/prod/dns_zone.tf

```diff
@@ -15,4 +15,9 @@ resource "azurerm_dns_zone" "apply-for-probate" {
 resource "azurerm_dns_zone" "appeal-benefit" {
   name                = "appeal-benefit-decision.service.gov.uk"
   resource_group_name = data.azurerm_resource_group.main.name
+}
+
+resource "azurerm_dns_zone" "divorce" {
+  name                = "divorce.service.gov.uk"
```

Comment:
i don't think this is a real zone, there's nothing in it.

I doubt this was actually delegated to us

Reply:
confirmed from devops team that this is a test record created. We can remove this zone altogether.